test/functional/webview-browser-tester-e2e-specs.js

```diff
@@ -0,0 +1,53 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import AndroidDriver from '../..';
+import path from 'path';
+import { sleep } from 'asyncbox';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+const AVD_ANDROID_24_WITHOUT_GMS = "Nexus_5_API_24";
+const capabilities = {
+  "browserName": "chromium-webview",
+  "avd": AVD_ANDROID_24_WITHOUT_GMS,
+  "platformName": "Android",
+  "platformVersion": "7.0",
+  "deviceName": "Android Emulator",
+  "chromedriverExecutable": path.join(process.cwd(), "chromedriver")
+};
+
+describe('Android 7 Webview Browser tester', function () {
+  let driver, test = this;
+  before(() => {
+    if (process.env.REAL_DEVICE) {
+      test.pending = true;
```

Comment:
This can more easily be written without the `=>` syntax (so we get the normal scoping) and getting rid of the `test` variable:
```js
before(function () {
  this.pending = process.env.REAL_DEVICE;
});
```
But also, what happens if it _is_ pending? Does the test not run?